images/js/googleCloud.js

```diff
@@ -0,0 +1,37 @@
+const { Storage } = require('@google-cloud/storage');
+
+const bucketName = 'gs://lottie-animation-community-tests';
+let storage;
+
+const initialize = () => {
+  try {
+    const key = process.env.GOOGLE_CLOUD_STORAGE;
+    const keyString = Buffer.from(key, 'base64').toString('ascii');
+    const storageKey = JSON.parse(keyString);
+    storage = new Storage({
+      credentials: storageKey,
+      projectId: storageKey.project_id,
+    });
+  } catch (error) {
+    console.log('Could not initialize Google Cloud');
```

Comment:
These could be consolidated into: 

 console.log('Could not initialize Google Cloud:', error);

Reply:
changed!